deployments/constructs/backing-services/istio.ts

```diff
@@ -35,31 +35,23 @@ export class Istio extends Construct {
       },
     );
 
-    const istioBase = new helm.release.Release(
-      this,
-      `istio-base${config.idPostfix}`,
-      {
-        dependsOn: [...config.dependsOn],
-        wait: true,
-        waitForJobs: true,
-        provider: config.helmProvider,
-        name: "istio-base",
-        chart: "base",
-        namespace: istioNamespace.metadata.name,
-        repository: "https://istio-release.storage.googleapis.com/charts",
-      },
-    );
+    const istioBase = new helm.release.Release(this, `istio-base-${name}`, {
+      dependsOn: [...config.dependsOn],
+      provider: config.helmProvider,
+      name: "istio-base",
+      chart: "base",
+      namespace: istioNamespace.metadata.name,
+      repository: "https://istio-release.storage.googleapis.com/charts",
```

Comment:
Is the version defined somewhere or does it deploy latest?
I'd prefer to pin down versions to have reproducible deployments

Reply:
this applies to all deployments

Reply:
all helm repository releases now have versions specified